### PR TITLE
feat(sdk): updated cloud.function runtime to nodejs18.x

### PR DESCRIPTION
**Pull request by garysassano**
Closes https://github.com/winglang/wing/issues/2134

*By submitting this pull request, I confirm that my contribution is made under the terms of the [Monada Contribution License](https://docs.winglang.io/terms-and-policies/contribution-license.html)*.

### CODE REVIEW

**Comment by garysassano** [comment]
Hurray! :satisfied:

**Comment by revitalbarletz** [comment]
Great job @garysassano! So proud of you :)

File: .devcontainer/devcontainer.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
```
Comment (hasanaburayyan) [inline .devcontainer/devcontainer.json:1]: This should not be commited to repo

**Review by hasanaburayyan — CHANGES_REQUESTED** [review]

File: .devcontainer/devcontainer.json (hunk 1 above)
Comment (garysassano) [reply .devcontainer/devcontainer.json]: Fixed that.

**Review by Chriscbr — APPROVED** [review]

**Review by hasanaburayyan — APPROVED** [review]

**Pull request merged**